### DISCUSSION TITLE
HCG, Proton-GE, DXVK_ASYNC and me

### DISCUSSION THREAD

**Discussion by hrvylein** [Heroic-Games-Launcher/HeroicGamesLauncher#1780] [Q&A]
Can anyone lead me how to properly activate DXVK_ASYNC in HCG on Steam Deck to reduce shader cache stuttering? I added "DXVK_ASYNC" with value "1" in ENV-Variables while using Proton-GE-7.24. But there is no difference without using that command. Are there additional steps needed? The "install/update dxvk on prefix"-option did nothing for me while editing the config of an already installed game.

Any help would be really appreciated.

**Comment by flavioislima** [comment]
Haven't tested this feature yet but if you can ser the Install DXVK on prefix it is because you are using Wine and not Proton.
Have you tried it with Proton Experimental as well?

**Reply by flavioislima** [reply]
Can you send a screenshot of the game config, game name etc. So I can try it as well?
In my case, when using Proton-GE, Wine-GE or Proton Experimental I cannot notice any stutter since for a long time now, I wonder if that is not enabled for all games already or also won't work for all of them.
